.github/workflows/dlp-pipelines.yml

```diff
@@ -362,6 +437,7 @@ jobs:
       - generate-uuid
       - inspection
       - de-identification
+      - de-identification-streaming-write
```

Comment:
Is table created in 'de-identification-streaming-write' being cleaned-up here?

Reply:
Yes because ultimately the whole dataset is being removed